charts/group-sync-operator/values.yaml

```
@@ -0,0 +1,18 @@
+acm-operatorpolicy:
+  subscription:
+    channel: alpha
+    name: group-sync-operator
+    namespace: group-sync-operator
+    source: community-operators
+    sourceNamespace: openshift-marketplace
+    startingCSV: group-sync-operator.v0.0.31
```

When using the latest version of ACM operatorpolicy chart, you can use parameter forceReleaseNamespace to deploy the operator to the helm release namespace, i.e. the namespace specified in conf repo versions.yaml, which is more flexible.

```suggestion
  subscription:
    channel: alpha
    name: group-sync-operator
    forceReleaseNamespace: true
    source: community-operators
    sourceNamespace: openshift-marketplace
    startingCSV: group-sync-operator.v0.0.31
```